examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -0,0 +1,225 @@
+/**
+ * Copyright 2014-2015 Axel Huebl, Alexander Debus
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/** Load pre-defined templates */
+#include "fields/background/templates/TWTS/TWTS.hpp"
+
+#ifndef PARAM_INCLUDE_FIELDBACKGROUND
+#define PARAM_INCLUDE_FIELDBACKGROUND false
+#endif
+
+/** Load external background fields
+ *
+ */
+namespace picongpu
+{
+    class FieldBackgroundE
+    {
+    public:
+    
+        /* Add this additional field for pushing particles */
+        static const bool InfluenceParticlePusher = PARAM_INCLUDE_FIELDBACKGROUND;
+
+        /* We use this to calculate your SI input back to our unit system */
+        const float3_64 unitField;
```

Comment:
please use `PMACC_ALIGN` to avoid side effects.

`PMACC_ALIGN(unitField,const float3_64);`